docs/mkdocs/docs/tutorials/lmdb_and_in_memory.md

```diff
@@ -0,0 +1,273 @@
+# LMDB and In-Memory storage backends
+
+ArcticDB can use a file-based LMDB backend, or a RAM-based in-memory storage, as alternatives to object storage such as S3 and Azure.
+
+Two simple examples are shown below.
+
+Note that this example writes to `test.lmdb` in the working directory.
+
+```py
+from arcticdb import Arctic
+import pandas as pd
+
+ac_lmdb = Arctic('lmdb://test.lmdb')
+lib = ac_lmdb.get_library('lib', create_if_missing=True)
+lib.write('symbol', pd.DataFrame({'a': [1, 2, 3]})
+print(lib.read('symbol').data)
+```
+
+which gives:
+
+```py
+   a
+0  1
+1  2
+2  3
+```
+
+The equivalent instantiation for an in-memory store uses the URI `'mem://'` as in:
+
+```py
+ac_mem = Arctic('mem://')
+```
+
+The storage returned is owned by the `ac_mem` instance and lives for the lifetime of this Python object. Behind the
+scenes, a [Folly Concurrent Hash Map](https://github.com/facebook/folly/blob/main/folly/concurrency/ConcurrentHashMap.h)
+is used as the key/value store. For test cases and experimentation, the in-memory backend is a good option.
+
+On the other hand, if the datasets are too large for RAM, or multiple processes want concurrent access to the same data
+then LMDB is recommended.
+
+## How to handle concurrent writers?
```

Comment:
I think we should focus this page on using ArcticDB with in-memory backends. Hence below we should be showing how to set up a tmpfs and run ArcticDB over it, and demonstrating that concurrent access works. I think we can skip the section describing LMDB over an on-disk filesystem.

Reply:
I've kept the snippet for how to connect to an `lmdb` endpoint because I think this is useful for also explaining the `mem://` syntax afterwards. I've changed it to the tmpfs case though so it fits the title of the guide. For most users they will likely visit this page and leave once they see that they can use `mem://`, I'd imagine.